### PR TITLE
Convert documentation to readthedocs

### PR DESCRIPTION
**Pull request by morgantaschuk**
See the test at https://runscanner-test.readthedocs.io/

I wanted to try out readthedocs and Run Scanner seemed small enough to do so. There's no substantial changes to the actual documentation. 

mkdocs isn't as supportive of inline HTML as Jekyll is, so I converted a lot to Markdown. Mkdocs also automatically makes links for headers so I removed the <a id="" /> commands (which weren't fully supported by mkdocs anyway).

And it looks like my editor also removed all of the whitespace at the end of lines. :/

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -17,40 +10,36 @@ For Illumina support:
 * C++ build environment
 * jsoncpp
 
-Please ensure that your sequencer(s) are supported by Run Scanner. 
-Please refer to <a href="appendices.html#C">Appendix C: Sequencers Supported 
+Please ensure that your sequencer(s) are supported by Run Scanner.
+Please refer to <a href="appendices.html#C">Appendix C: Sequencers Supported
```
Comment (avarsava) [inline docs/installation.md]: Broken anchor link

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -1,107 +1,101 @@
...
+    Directories will appear in the Unreadable list if they do not contain files
+    which Run Scanner expects to find for a run. (For example, Illumina runs are
+    expected to contain _runParameters.xml_, and PromethION runs are expected to
+    contain a <i>fast5_pass</i> directory.) Ensure that the
+    directory contains run data.
+
+2.  **Verify format of run data.**
+    Run Scanner expects run data to be in certain formats per sequencer.
+    Please ensure your sequencer is supported and running a supported software
+    version. Please refer to <a href="appendices.html#C">Appendix C: Sequencers
```
Comment (avarsava) [inline docs/troubleshooting.md]: broken link

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -1,107 +1,101 @@
...
+being hosted must have read and execute permission for the run data in the
 sequencing output directory. Grant these permissions if necessary.
 
 #### Ensure runscanner-illumina is installed
-If runs from an Illumina sequencer are universally failing, ensure that 
-runscanner-illumina is installed correctly, as outlined in 
-<a href="illuminasetup.html">Illumina Setup</a>. Illumina runs 
+If runs from an Illumina sequencer are universally failing, ensure that
+runscanner-illumina is installed correctly, as outlined in
+<a href="illuminasetup.html">Illumina Setup</a>. Illumina runs
```
Comment (avarsava) [inline docs/troubleshooting.md]: broken link

File: docs/installation.md (hunk 1 above)
Comment (morgantaschuk) [reply docs/installation.md]: Fixed.

**Pull request merged**